tests/integration/suite/test_istio.py

```diff
@@ -66,6 +66,48 @@ def test_destination_rule(admin_pc):
     client.delete(ns)
 
 
+# consistentHash has an "anyOf" openAPI validation on it,
```

Comment:
do you mean oneOf? AnyOf would allow both to be filled in.

Reply:
dang, you're right

Reply:
fixed